server/routes/approvingVariations/handlers/comDetails.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from 'express'
+import CommunityService from '../../../services/communityService'
+
+export default class ComDetailsRoutes {
```

Comment:
Another duplicate of this page 😓 not a fault with this PR but something that would be good to sort as tech debt. I think a ticket already exists on the backlog